src/scripting_api/field.js

```diff
@@ -393,15 +394,32 @@ class Field extends PDFObject {
   }
 
   getArray() {
+    // Gets the array of terminal child fields (that is, fields that can have
+    // a value for this Field object, the parent field.
```

Comment:
Nit: missing closing parenthesis here.
```suggestion
    // a value for this Field object, the parent field).
```